src/main/proto/eventtemplates/video_template.proto

```diff
@@ -28,28 +28,25 @@ message Video {
     description: "An Event Template for video ad events."
   };
 
-  // Measures the percentage of a video ad that was viewed.
-  message DigitalVideoCompletionStatus {
-    bool completed_0_percent_plus = 1;
-    bool completed_25_percent_plus = 2;
-    bool completed_50_percent_plus = 3;
-    bool completed_75_percent_plus = 4;
-    bool completed_100_percent = 5;
-  }
+  // These field measure the percentage of a video ad that was viewed.
```

Comment:
This comment will be applied only to the field above which it is placed, meaning it applies only to the `completed_0_percent_plus` field.

Each field needs to be documented separately instead.

Reply:
Thanks. Hope this covers what we are looking for.